go/vt/vtorc/db/db.go

```diff
@@ -82,25 +77,12 @@ func deployStatements(db *sql.DB, queries []string) error {
 	tx, err := db.Begin()
 	if err != nil {
 		log.Fatal(err.Error())
+		return err
```

Comment:
@dbussink looked at this more - this line will never be reached due to the `log.Fatal(...)` before it

There is also another `log.Fatal(...)` in this func; it might be cleaner for `registerVTOrcDeployment` to only `return err`s _(no `log.Fatal`s)_ and `initVTOrcDB` can do a single `log.Fatal(...)` if `registerVTOrcDeployment` errors - then all the fatal-ing code is in the init function

Reply:
Ah yeah, this seems pretty inconsistent style wise at the moment, so I think that's more something for a separate PR to make consistent.

Reply:
Makes sense 👍